cgi-bin/LJ/Entry.pm

```diff
@@ -2045,6 +2045,70 @@ sub load_log_props2
 }
 
 # <LJFUNC>
+# name: LJ::load_talk_props2
```

Comment:
Since talkprops are all for comments, wouldn't these better go in `LJ::Comment`?